applications/zpc/components/zwave_command_classes/src/zwave_command_class_meter_control.c

```diff
@@ -247,9 +260,7 @@ static sl_status_t zwave_command_class_meter_supported_get(
     ZW_METER_GET_V5_FRAME *get_frame = (ZW_METER_GET_V5_FRAME *)frame;
```

Comment:
dont you want to update cast here too?

Reply:
Yes, to be pedantic, but structs are not packed, and V5 has few more extra elements to V1 so this will also work.